jobs/gcp-discovery/src/main/java/com/tmobile/pacbot/gcp/inventory/collector/APIKeysInventoryCollector.java

```diff
@@ -23,79 +24,80 @@ public class APIKeysInventoryCollector {
 
-               HashMap<String,Object>browserKeyRestrictions=new HashMap<>();
+                HashMap<String, Object> browserKeyRestrictions = new HashMap<>();
 
-               keys.getRestrictions().getBrowserKeyRestrictions().getAllFields().forEach((fieldDescriptor, o) -> {
-                   browserKeyRestrictions.put(fieldDescriptor.getName(),o);
-                   logger.info( fieldDescriptor.getName(),o);
+                keys.getRestrictions().getBrowserKeyRestrictions().getAllFields().forEach((fieldDescriptor, o) -> {
+                    browserKeyRestrictions.put(fieldDescriptor.getName(), o);
+                    logger.info(fieldDescriptor.getName(), o);
 
-               });
-               restriction.put("browserKeyRestrictions",browserKeyRestrictions);
+                });
+                restriction.put("browserKeyRestrictions", browserKeyRestrictions);
 
-               HashMap<String,Object>androidKeyRestrictions=new HashMap<>();
+                HashMap<String, Object> androidKeyRestrictions = new HashMap<>();
 
-               keys.getRestrictions().getAndroidKeyRestrictions().getAllFields().forEach((fieldDescriptor, o) -> {
-                   androidKeyRestrictions.put(fieldDescriptor.getName(),o);
-                   logger.info( " androidKeyRestrictions {}",o);
+                keys.getRestrictions().getAndroidKeyRestrictions().getAllFields().forEach((fieldDescriptor, o) -> {
+                    androidKeyRestrictions.put(fieldDescriptor.getName(), o);
+                    logger.info(" androidKeyRestrictions {}", o);
 
-               });
-               restriction.put("androidKeyRestrictions",androidKeyRestrictions);
+                });
+                restriction.put("androidKeyRestrictions", androidKeyRestrictions);
 
-               HashMap<String,Object>iosKeyRestrictions=new HashMap<>();
+                HashMap<String, Object> iosKeyRestrictions = new HashMap<>();
 
-               keys.getRestrictions().getIosKeyRestrictions().getAllFields().forEach((fieldDescriptor, o) -> {
-                   iosKeyRestrictions.put(fieldDescriptor.getName(),o);
-                   logger.info( " iosKeyRestrictions {}",o);
+                keys.getRestrictions().getIosKeyRestrictions().getAllFields().forEach((fieldDescriptor, o) -> {
+                    iosKeyRestrictions.put(fieldDescriptor.getName(), o);
+                    logger.info(" iosKeyRestrictions {}", o);
 
-               });
-               restriction.put("iosKeyRestrictions",iosKeyRestrictions);
-           }
+                });
+                restriction.put("iosKeyRestrictions", iosKeyRestrictions);
+            }
 
-           apiKeysVH.setCreatedDate(new Date(keys.getCreateTime().getSeconds()*1000).toString());
+            SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSSSSS");
+            apiKeysVH.setCreatedTime(dateFormat.format(new Date(keys.getCreateTime().getSeconds() * 1000)));
```

Comment:
This is the format used at CQ mapper file. This is also in consistent with the format used in policy RotateAPIKeysRule.